tests/integration/single_table/test_single_table.py

```diff
@@ -17,6 +18,8 @@
     KSTestExtended,
     LogisticDetection,
     SVCDetection,
+    RandomForestClassifierDetection,
+    GradientBoostingClassifierDetection
```

Comment:
I believe you are missing a comma on this line.
```
E     File "/home/runner/work/SDMetrics/SDMetrics/tests/integration/single_table/test_single_table.py", line 23
E       ContinuousKLDivergence,
E                            ^
E   SyntaxError: invalid syntax
```

Reply:
I fixed the style issues.